WooCommerce/Classes/POS/Analytics/WooAnalyticsEvent+PointOfSale.swift

```diff
@@ -50,6 +58,12 @@ extension WooAnalyticsEvent {
                 Key.checkoutTapCount: "\(checkoutTapCount)"
             ])
         }
+
+        static func cashCollectPaymentSuccess(millisecondsSinceCustomerIteractionStarted: Double) -> WooAnalyticsEvent {
+            WooAnalyticsEvent(statName: .pointOfSaleCashCollectPaymentSuccess, properties: [
+                Key.millisecondsSinceCustomerInteractionStarted: "\(millisecondsSinceCustomerIteractionStarted)",
```

Comment:
Theoretically we could also track `millisecondsSinceOrderCreationSuccess` but maybe it's not needed.